### ISSUE TITLE
Add to home assistant? Or give examples?

### ISSUE THREAD

**Issue by jschwalbe** [woodsbw/gabb#3]
Hi,

It looks like you've put in a good amount of effort to get here. Any interest in explaining how to use it and/or helping to integrate it with Home Assistant?

I would love to use this; I'm quite comfy at the command line, but I'm afraid not with deciphering python.

Thanks much!

**Comment by woodsbw** [comment]
Hey there. Actually, that was my original intention before life got to crazy to finish it.

Unfortunately, I never got the chance to wrap back around and figure out enough about the HA plug-in infrastructure to get it done. So, I could advise on how to use the library, but no really on the HA side.

**Comment by slackcc** [comment]
@woodsbw I'm going to see if I can do anything about getting this working with HA. One question, where did you find the paths for accessing everything? Do you have access to a developers guide for the filip-pass stuff? Just trying to see if there's any way to access the current battery life, it shows up in the parent app so I'm guessing there must be an API query that can be done at the device level, but doesn't look like that's implemented anywhere in the client.

**Comment by woodsbw** [comment]
Unfortunately, no dev guide (that I have access to, at least.) This was all reserve engineered by forcing the traffic through a proxy and watching the calls that were made.

**Comment by slackcc** [comment]
Yeah, figured as much, do you still have the proxy setup? If you found the battery path I'd be happy to do a PR and add a method to the client to allow it to pull that info.

**Comment by jaycollett** [comment]
I'd be interested in helping out here, I just built a Python script to use the library and dump out the JSON data for each of the devices in my account (multiple kids). Was just going to head down the path of figuring out HA integrations...

**Comment by jaycollett** [comment]
https://myfilip.smartcom-utils.com/ seems like Gabb has repurposed a whole lot....

**Comment by jaycollett** [comment]
Not a true HA integration, but I use MQTT already, and it seems to work great.  
https://github.com/jaycollett/gabbwireless_mqtt

**Comment by slackcc** [comment]
@jaycollett Nice! I'm going to give that a try. At least I will if I can get my son's Gabb watch to work again, it just started randomly resetting like 20 times a day a couple weeks ago.
